### DISCUSSION TITLE
After several crashes Workshop won't start

### DISCUSSION THREAD

**Discussion by 1800joe** [noop-inc/documentation#68] [Bugs] [answered]
<img width="1392" alt="Screenshot 2024-01-12 at 11 17 42 AM" src="https://github.com/noop-inc/documentation/assets/3111670/ac907691-f7dc-4cd3-8fd2-9b416c85e33a">

Loading for >10 minutes

**Comment by arbitrarytech** [comment]
This is likely due to out of memory issues in the Workshop VM that we are planning to address in the next release.

**Comment by 1800joe** [comment, marked as answer]
Appears resolved in v0.5.x
